### PR TITLE
Update knoxAutoPolicy build scripts to use kubearmor docker repository

### PR DESCRIPTION
**Pull request by seswarrajan**
Signed-off-by: Eswar Rajan Subramanian [eswar@accuknox.com](mailto:eswar@accuknox.com)

### CODE REVIEW

File: .github/workflows/latest-release.yml [hunk 1]
```diff
@@ -75,8 +75,8 @@ jobs:
         run: |
             value=`cat STABLE-RELEASE`
             if [ ${{ github.ref }} == refs/heads/$value ]; then
-              docker pull docker.io/accuknox/knoxautopolicy:$value
-              docker image accuknox/knoxautopolicy:$value accuknox/knoxautopolicy:stable
+              docker pull docker.io/kubearmor/knoxautopolicy:$value
+              docker image kubearmor/knoxautopolicy:$value kubearmor/knoxautopolicy:stable
```
Comment (nyrahul) [inline .github/workflows/latest-release.yml:79]: lets keep the docker hub repo to accuknox. If we change this then we have to change the github repo secrets as well.

File: .github/workflows/stable-release.yml [hunk 2]
```diff
@@ -22,8 +22,8 @@ jobs:
       - name: Pull Docker Hub image
         run: |
             value=`cat STABLE-RELEASE`
-            docker pull docker.io/accuknox/knoxautopolicy:$value
-            docker image tag accuknox/knoxautopolicy:$value accuknox/knoxautopolicy:stable
+            docker pull docker.io/kubearmor/knoxautopolicy:$value
+            docker image tag kubearmor/knoxautopolicy:$value kubearmor/knoxautopolicy:stable
```
Comment (nyrahul) [inline .github/workflows/stable-release.yml:26]: revert this

File: deployments/k8s/deployment.yaml [hunk 3]
```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       serviceAccountName: knoxautopolicy
       containers:
-      - image: accuknox/knoxautopolicy:stable
+      - image: kubearmor/knoxautopolicy:stable
```
Comment (nyrahul) [inline deployments/k8s/deployment.yaml:18]: revert this

File: src/build/push_knoxautopolicy.sh [hunk 4]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 AUTOPOL_HOME=`dirname $(realpath "$0")`/../..
-[[ "$REPO" == "" ]] && REPO="accuknox/knoxautopolicy"
+[[ "$REPO" == "" ]] && REPO="kubearmor/knoxautopolicy"
```
Comment (nyrahul) [inline src/build/push_knoxautopolicy.sh:4]: revert this

File: src/build/build_knoxautopolicy.sh [hunk 5]
```diff
@@ -3,7 +3,7 @@
 AUTOPOL_HOME=`dirname $(realpath "$0")`/../..
 cd $AUTOPOL_HOME
 AUTOPOL_SRC_HOME=$AUTOPOL_HOME/src
-[[ "$REPO" == "" ]] && REPO="accuknox/knoxautopolicy"
+[[ "$REPO" == "" ]] && REPO="kubearmor/knoxautopolicy"
```
Comment (nyrahul) [inline src/build/build_knoxautopolicy.sh:6]: revert this

**Review by nyrahul — CHANGES_REQUESTED** [review]

**Comment by seswarrajan** [comment]
Currently using the same docker repository - accuknox
Hence closing the PR

**Pull request closed without merging**